lib/recursive-methods.rb

```diff
@@ -30,16 +40,27 @@ def nested(s)
     raise NotImplementedError, "Method not implemented"
 end
 
-# Time complexity: ?
-# Space complexity: ?
+# Time complexity: O(n)
+# Space complexity: O(n^2)
 def search(array, value)
```

Comment:
👍 However the time complexity is O(n^2) because you're making a copy of the original array with each iteration.